polynote-kernel/src/test/scala/polynote/kernel/lang/KernelSpec.scala

```diff
@@ -37,13 +37,13 @@ trait KernelSpec {
   }
 
   def assertPythonOutput(code: Seq[String])(assertion: (Map[String, Any], Seq[Result], Seq[(String, String)]) => Unit): Unit = {
-    assertOutputWith((kernelContext: KernelContext, updates) => PythonInterpreter.factory()(kernelContext, MockVenvDepProvider(updates)), code) {
+    assertOutputWith((kernelContext: KernelContext, updates) => PythonInterpreter.factory()(kernelContext, MockVenvDepProvider(updates)).unsafeRunSync(), code) {
```

Comment:
Not terribly important, but if we  make `mkInterp` in `assertOutputWith` return an `IO[K]` we could probably fold it all together and not need this `unsafeRunSync` call.